packages/react/src/components/NumberInput/NumberInput-story.js

```diff
@@ -28,6 +35,7 @@ const props = () => ({
   max: number('Maximum value (max)', 100),
   value: number('Value (value)', 50),
   step: number('Step of up/down arrow (step)', 10),
+  size: select('Field size (size)', sizes, undefined) || undefined,
```

Comment:
```suggestion
  size: select('Field size (size)', sizes, undefined),
```

do we still need this part?

Reply:
Not sure, we can make another PR to remove them from each of the size examples though if it is not needed.

Reply:
Seems like this is still needed. If you swap to a different size and then swap back to default, I'm seeing a console warning.